### PR TITLE
Use approved wording for GCP PSC

### PR DESCRIPTION
**Pull request by simnyc**
* replace "GCP Private Service Connect" with "Google Cloud Private Service Connect"
* replace "GCP" with "Google Cloud" 

> we are going with "Google Cloud Private Service Connect" uniformly for the integration for ease of use in finding it and it also is more consistent with our other naming

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by sriramkraman — APPROVED** [review]
LGTM

File: content/en/agent/guide/gcp-private-service-connect.md [hunk 1]
```diff
@@ -15,21 +15,21 @@ further_reading:
...
+[Google Cloud Private Service Connect][1] (PSC) allows you to send telemetry to Datadog without using the public internet.
 
-Datadog exposes some of its data intake services in Google Cloud Platform as PSC [_published services_][2], as seen in the [table of published services](#published-services). 
+Datadog exposes some of its data intake services in Google Cloud as Private Service Connect [_published services_][2], as seen in the [table of published services](#published-services). 
 
-You can configure a GCP PSC endpoint to expose a private IP address for each Datadog intake service; this IP address routes traffic to the Datadog backend. You can then configures a GCP [_Private DNS Zone_][3] to override the DNS names corresponding to the products for each endpoint that is consumed. 
+You can configure a PSC endpoint to expose a private IP address for each Datadog intake service; this IP address routes traffic to the Datadog backend. You can then configures a Google Cloud [_Private DNS Zone_][3] to override the DNS names corresponding to the products for each endpoint that is consumed.
```
Comment (drichards-87) [inline content/en/agent/guide/gcp-private-service-connect.md:22]:
```suggestion
You can configure a PSC endpoint to expose a private IP address for each Datadog intake service. This IP address routes traffic to the Datadog backend. You can then configure a Google Cloud [_Private DNS Zone_][3] to override the DNS names corresponding to the products for each endpoint that is consumed. 
```

File: content/en/agent/guide/gcp-private-service-connect.md [hunk 2]
```diff
@@ -126,17 +126,17 @@ Ensure that the IP address in the response matches the one associated with your
 
 Datadog exposes some of its data intake services in Google Cloud Platform as PSC [_published services_][2], as seen in the [table of published services](#published-services). 
 
-You can configure a GCP PSC endpoint to expose a private IP address for each Datadog intake service; this IP address routes traffic to the Datadog backend. You can then configures a GCP [_Private DNS Zone_][3] to override the DNS names corresponding to the products for each endpoint that is consumed. 
+You can configure a PSC endpoint to expose a private IP address for each Datadog intake service; this IP address routes traffic to the Datadog backend. You can then configures a Google Cloud [_Private DNS Zone_][3] to override the DNS names corresponding to the products for each endpoint that is consumed.
```
Comment (drichards-87) [inline content/en/agent/guide/gcp-private-service-connect.md:129]:
```suggestion
You can configure a PSC endpoint to expose a private IP address for each Datadog intake service. This IP address routes traffic to the Datadog backend. You can then configure a Google Cloud [_Private DNS Zone_][3] to override the DNS names corresponding to the products for each endpoint that is consumed. 
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left two very small suggestions.

**Comment by drichards-87** [comment]
@simnyc Are you still waiting on a review from @shris-github?

**Review by drichards-87 — APPROVED** [review]

**Comment by simnyc** [comment]
> @simnyc Are you still waiting on a review from @shris-github?

no actually i used the wrong GH handle, you can ignore this review request and merge this. thanks

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**